CHANGES.md

```diff
@@ -1,6 +1,16 @@
 emoji
 =====
 
+
+1.1.0
+-----
+* Added support of Italian naming of emojis
+
+1.0.1
+-----
+* Added Python 3.8 and 3.9 as supported versions
+* Minot bug fixing
```

Comment:
I assume this is a typo of "Minor", but #147 was hardly minor, since it caused any import of the library itself to error.